app/reducers/index.jsx

```diff
@@ -2,6 +2,7 @@ import { combineReducers } from 'redux'
 
 const rootReducer = combineReducers({
   auth: require('./auth').default,
+  product: require('./product').default
```

Comment:
I actually already wrote this in a different merge, so delete this so we don't get any merge conflicts!